### PR TITLE
docs(blog): run Ibis on Snowflake

### PR DESCRIPTION
**Pull request by IndexSeek**
This PR aims to create a post to tell the story about how we were able to get Ibis running inside Snowflake, fulfilling the reference in  https://github.com/ibis-project/ibis/blob/bdc1b3fbb8d44f9246c957b58372d3df84247468/docs/posts/ibis-version-9.0.0-release/index.qmd#L135-L136

This all ties back to #5877, with the resolution made possible thanks to the effort from converting to SQLGlot from SQLAlchemy.

### CODE REVIEW

**Comment by IndexSeek** [comment]
This is still a WIP, but I wanted to get this out here so that we can collaborate via PR and ensure the structure of the post is on the right track.

**Comment by cpcloud** [comment]
@IndexSeek Thanks!

I pushed up some changes explaining how the thing works, linking to various places for more information.

**Comment by IndexSeek** [comment]
Thank you @cpcloud! These modifications and additions were great. 

I made some slight tweaks to the phrasing around the more permanent solution. I want readers to know that it is possible, but I want to avoid making any promises or setting expectations that this would be made available directly from Ibis.

**Comment by IndexSeek** [comment]
As I reread this, I realized that the note I had included: "Now we can upload these packages and make them available" was abrupt and lacked clarity on why it's necessary. I included a small snippet showing the sproc registration to include the other dependencies as well, which may encourage readers to try it themselves. 

I am happy to tweak this if it's too much code/too specific.

**Review by cpcloud — APPROVED** [review]
This LGTM!

File: docs/posts/run-on-snowflake/index.qmd [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+functions and stored procedures.
+
+If users could use Ibis directly within the platform, developers could more
+easily switch between a local execution engine during development and
+efficiently deploy and operationalize that same code on Snowflake.
+
+But this isn't without its challenges; there were a few things we needed to
+figure out, and these are the questions we will answer throughout the post.
+
+- How can we get an Ibis connection to Snowflake - from in Snowflake?
```
Comment (gforsyth) [inline docs/posts/run-on-snowflake/index.qmd:35]:
```suggestion
- How can we get an Ibis connection to Snowflake - from within Snowflake?
```

or `inside`, maybe?

**Review by gforsyth — APPROVED** [review]
Looks good!  One incredibly small, non-blocking language nit.

File: docs/posts/run-on-snowflake/index.qmd (hunk 1 above)
Comment (IndexSeek) [reply docs/posts/run-on-snowflake/index.qmd:35]: I like the usage of "within" here. That is a bit clearer than using "from in" together.  

Thank you for the review!

**Comment by cpcloud** [comment]
@IndexSeek Once you're ready to ship this, please mark it as Ready for review, we'll merge it!

**Pull request merged**